src/tester/actions/BackAction.js

```diff
@@ -22,7 +21,7 @@ export default class BackAction extends AbstractAction {
     static async isActionAvailable(element, page) {
         const historyLength = await page.evaluate(() => window.history.length);
 
-        return historyLength > 2 && Math.floor(Math.random() * 100) <= BACK_CHANCE;
+        return historyLength > 2 && Math.floor(Math.random() * 100) >= BACK_CHANCE;
```

Comment:
The original line was actually correct. I guess this resulted from bad conflict resolution with master branch.